src/common/services/api/designations.service.js

```diff
@@ -268,7 +268,8 @@ class DesignationsService {
 
   ministriesList (pagePath) {
     const path = pagePath + '/jcr:content/content-parsys/designation_search_r.json'
-    return Observable.from(this.$http.get(this.$location.origin + '/' + path))
+    const url = `${this.$location.protocol()}://${this.$location.host()}/${path}`
```

Comment:
If the port will ever be different from 80, `$location.host()` doesn't include the port.

Reply:
Yeah, I was debating on including port. It will only matter on local machines, which only I have a full local environment set up with a locally running AEM and front-end web at this point.